README.md

```diff
@@ -17,10 +17,10 @@ You’ll need a Kubernetes cluster to run against. You can use [KIND](https://si
 make deploy
 ```
 
-2. Create an IstioHelmInstall instance to install istiod:
+2. Create an instance of the Istio resource to install the Istio Control Plane:
 
 ```sh
-kubectl apply -f config/samples/maistra.io_v1_istiohelminstall.yaml
+kubectl apply -f config/samples/operator.istio.io_v1alpha1_istio.yaml
```

Comment:
not related to this pr, but... we could rename this file to something simpler, don't you think?

Reply:
I think we can. I can do it in a follow-up PR, where I'll also rename a few other things (e.g. `istio-editor-role` to `istio-editor`).